src/client/features/view/components/menu/search.js

```diff
@@ -0,0 +1,96 @@
+const _ = require('lodash');
+
+//Apply hover styling to a collection of nodes
+function updateStyling(style, matched, cy) {
+  _.forEach(matched, node => {
+    let isCompartment = node.data('class') === 'compartment';
+
+    //Collapse Expanded Targets
+    if (node.isExpanded() && !isCompartment) { node.collapse(); }
+
+    //Apply style to the parent
+    if (node.isChild() && node.parent().data('class') !== 'compartment') { node = node.parent(); }
+
+    applySearchStyle(cy, node, style);
```

Comment:
Remove cy from the applySearchStyle params

Reply:
Cy has been removed